source/FreeRTOS_DHCPv6.c

```diff
@@ -177,7 +177,7 @@ static void prvCreateDHCPv6Socket( NetworkEndPoint_t * pxEndPoint );
 static void prvCloseDHCPv6Socket( NetworkEndPoint_t * pxEndPoint );
 
 #if ( ipconfigHAS_DEBUG_PRINTF == 1 )
-    static const char * prvStateName( eDHCPState_t eState );
+    const char * prvStateName( eDHCPState_t eState );

```

Comment:
Why is `static` removed here? I don't see any other files calling this function.